dotnet/src/SemanticKernel.Abstractions/Functions/ParameterView.cs

```diff
@@ -13,9 +16,13 @@ namespace Microsoft.SemanticKernel;
 /// <param name="DefaultValue">Default parameter value, if not provided</param>
 /// <param name="Type">Parameter type.</param>
 /// <param name="IsRequired">Whether the parameter is required.</param>
+/// <param name="NativeType">The native type. Null if this parameter did not come from a native function.</param>
+/// <param name="Schema">The JSON Schema of the type. May be null for native function parameters.</param>
 public sealed record ParameterView(
     string Name,
     string? Description = null,
     string? DefaultValue = null,
     ParameterViewType? Type = null,
-    bool? IsRequired = null);
+    bool? IsRequired = null,
+    Type? NativeType = null,
```

Comment:
How about using `ParameterType` to be consistent with reflection API.

https://learn.microsoft.com/en-us/dotnet/api/system.reflection.parameterinfo.parametertype?view=net-7.0